packager/app/mpd_flags.cc

```diff
@@ -75,3 +75,10 @@ DEFINE_bool(dash_force_segment_list,
             "content is huge and the total number of (sub)segment references "
             "is greater than what the sidx atom allows (65535). Currently "
             "this flag is only supported in DASH ondemand profile.");
+DEFINE_bool(is_low_latency_dash,
```

Comment:
Proposed flag description:

"If enabled, low latency DASH (LL-DASH) will be used. This will reduce overall latency by ensuring that the media segments are chunk encoded and delivered via an aggregating response. The combination of these features will ensure that overall latency can be decoupled from the segment duration. For low latency to be achieved, the output of the packager must be combined with a delivery system which can chain together a set of aggregating responses (such as chunked transfer encoding under HTTP/1.1 or a HTTP/2 or HTTP/3 connection), as well as a DASH client which understands availabilityTimeOffset and the throughput estimation and ABR challenges which arise when operating in the low latency regime."

Reply:
Moving the thorough description to docs/source/options/dash_options.rst (https://google.github.io/shaka-packager/html/tutorials/dash.html) and will include usage examples.